### ISSUE TITLE
Install fails OS X 10.10

### ISSUE THREAD

**Issue by Bobspadger** [mkleehammer/pyodbc#55]
I've tried installing as Me, Sudo and Sudo -H but getting the following error:

```
Command "/Library/Frameworks/Python.framework/Versions/3.4/bin/python3.4 -c "import setuptools, tokenize;__file__='/private/tmp/pip-build-av0bnl1e/pyodbc/setup.py';exec(compile(getattr(tokenize, 'open', open)(__file__).read().replace('\r\n', '\n'), __file__, 'exec'))" install --record /tmp/pip-q3wvh5x7-record/install-record.txt --single-version-externally-managed --compile" failed with error code 1 in /private/tmp/pip-build-av0bnl1e/pyodbc
```

I have it working for python2 , but python3 seems to be a bit sticky

Any help much appreciated :)

**Comment by Bobspadger** [comment]
Any helpers ?

**Comment by tekknolagi** [comment]
What error did you get/

**Comment by mkleehammer** [comment]
I assume you are using pip.  Can I get a log file from it.  Run it again but add ``--log issue55.log``

Thanks.

**Comment by Bobspadger** [comment]
I eventually managed to get this to install but I can't remember how
I asked on stack exchange and got this:
http://stackoverflow.com/questions/31242358/pyodbc-3-0-10-fails-to-install-os-x-10-10

**Comment by mkleehammer** [comment]
I'm closing this.  I've released new binary wheels for version 4.0 and that should help.
